src/components/WishList/WishList.tsx

```diff
@@ -1,19 +1,22 @@
-import * as React from 'react';
+import React, { useContext } from 'react';
 import { Grid } from '@mui/material';
+import FavoriteContext from '../../store/Favorite/favorite-context';
 import WishListItem from './WishListItem/WishListItem';
+import { CartItem } from '../../store/types';
 
 interface WishListProps {
-  favoriteItems: FavoriteItem[];
+  favoriteItems: CartItem[];
 }
 
-interface FavoriteItem {
-  id: string;
-  name: string;
-  img: string;
-  price: number;
-}
+const WishList: React.FC<WishListProps> = () => {
+  const {
+    handleAddFavoriteItem,
+    handleIncrementFavoriteItem,
+    handleRemoveFavoriteItem,
+    handleDecrementFavoriteItem,
```

Comment:
Where do you use it?

Reply:
For adding Products from wish list to order n the same way as in ProductItem Component